eos/utils/expression-visitors.cc

```diff
@@ -246,6 +246,14 @@ namespace eos::exp
     {
     }
 
+    std::string
+    ExpressionCloner::next_hidden_alias()
+    {
+        this->_hidden_alias_index++;
+
+        return stringify(_hidden_alias_index);
```

Comment:
This produces a number as the name of the hidden kinematic variable. Does it make sense to name the variables, e.g., ``internal_variable_{idx}``?